scripts/docker_build.sh

```diff
@@ -17,8 +18,17 @@
 # SPDX-License-Identifier: Apache-2.0
 # License-Filename: LICENSE
 
+# Docker from ORT uses buildkit feature
```

Comment:
Change buildkit to BuildKit which is the official name per https://docs.docker.com/develop/develop-images/build_enhancements/